scripts/executable.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+function cleanup {
+  rm -rf build
+  rm AppleDevIDApp.p12
+  rm notarize_config.json
+  security delete-keychain percy.keychain
+}
+
+npm install -g pkg
+
+yarn install
+yarn build
+
+# Remove type from package.json files
+sed -i '' '/"type": "module",/d' ./package.json
```

Comment:
don't fail silently, if match not found.

Reply:
sed does not exit 1 even if match is not found.

Reply:
you'll need to use GNU sed - [ref](https://stackoverflow.com/a/66365284/9485283) 

```bash
➜  echo "Bash Scripting Language" | gsed 's/Bash/Perl/;tx;q1;:x'
Perl Scripting Language
➜  echo $?
0
➜  echo "Bas Scripting Language" | gsed 's/Bash/Perl/;tx;q1;:x'
Bas Scripting Language
➜  echo $?
1
```

Reply:
updated